### ISSUE TITLE
[Bug] Exiting content in fullscreen

### ISSUE THREAD

**Issue by veckia9x** [MediaBrowser/Emby#3622]
I am using Emby 4.4.2.0 on docker, latest Chrome, latest Windows 10 upgrade.

When I'm viewing a content and put it in fullscreen it doesnt exit the fullsxreen mode if I just click in the go back arrow on the top left inside emby, I need to do it manually using F11 or start a new content then click on fullscreen button again, this happens to all of then, tv show, movies, dvr, personal, etc...

Did a quick search on issues here, found nothing related.

**Comment by LukePulverenti** [comment]
Hi there, can you try 4.4.3 and see how that compares? Thanks.

**Comment by veckia9x** [comment]
Yes I can, I'll need some time to do it though. I can't work on it right know, wife would kill me. lol

**Comment by veckia9x** [comment]
Yes, problem persists on newer version (4.4.3.0).
I use the official docker container (emby/embyserver) but this one doesnt have the newer version, so I used the second best container (linuxserver/emby) wich is on 4.4.3.0.

**Comment by veckia9x** [comment]
Updated from the official docker, same issue still happening.

**Comment by veckia9x** [comment]
Updated to version 4.5.1.0, issue still happening.

**Comment by veckia9x** [comment]
Issue still happening.

Client specs:
- Windows 10 Home (version 2004)
- Chrome (87.0.4280.88) 64bits
- i5-5675C @ stock
- 8GB DDR3
- SSD NVME 480GB

Server specs:
- Emby Server Oficial Docker (Version 4.5.4.0)
- Emby premiere.
- Quicksync transcode enabled.
- i3-4150T
- 4GB DDR3L
- SSD NVME 128GB
- A few HDDs
